app/assets/javascripts/interactive-sizing.js

```diff
@@ -4,16 +4,31 @@ function interactiveSizing () {
   function setSize () {
     var $iframe = $(this);
     var aspectRatio = $iframe.data('aspect-ratio');
-    var currWidth = $iframe.width()
-    $iframe.attr('width', '100%');
-    $iframe.attr('height', currWidth / aspectRatio);
+    var resizeMethod = $iframe.data('aspect-ratio-method');
+    var magicNumber = 125;
```

Comment:
Different themes could change the size of the header or other things. Is this magic number necessary? Is it the same way the pinning code computes the maxHeight?